proto/interchain_security/ccv/provider/v1/provider.proto

```diff
@@ -464,3 +464,20 @@ message PowerShapingParameters {
 // ConsumerIds contains consumer ids of chains
 // Used so we can easily (de)serialize slices of strings
 message ConsumerIds { repeated string ids = 1; }
+
+// ConsumerPhase indicates the phases of a consumer chain according to ADR 018
+enum ConsumerPhase {
+  // UNSPECIFIED defines an empty phase.
+  CONSUMER_PHASE_UNSPECIFIED = 0;
```

Comment:
Why do we repeat `CONSUMER_PHASE` in here and then everything ends up being `ConsumerPhase_CONSUMER_PHASE_INITIALIZED` having consumer phase twice? Why not call this `UNSPECIFIED`, `INITIALIZED`, etc.?

Reply:
This is how is done in the SDK https://github.com/cosmos/cosmos-sdk/blob/355f4d7e422f111b6a61243872adc6d5661f4cad/x/gov/proto/cosmos/gov/v1/gov.proto#L17